TLM/TLM/UI/ModUI.cs

```diff
@@ -107,11 +107,11 @@ public class UIOpacityObservable : GenericObservable<UIOpacityNotification> {
 
         public void Destroy() {
             Log._Debug("ModUI destructor is called.");
-            Destroy(MainMenuButton);
-            Destroy(MainMenu);
+            DestroyImmediate(MainMenuButton);
```

Comment:
Why isn't Destroy good enough?

Reply:
I'm pretty sure it is good enough (pretty low chance for destroying it and instantiating new components within the same frame)

Reply:
pretty high chance during hot-reload. two versions of TMPE menu will be alive together. I hid the previous TMPE menu in my original Hot reload PR to avoid confusion. But now that I know of Destroy Immediate I think it is a good idea.